.claude/skills/run-ci-tests/bin/ci-env.py

```
@@ -0,0 +1,98 @@
+    for i, l in enumerate(lines):
+        if re.match(rf"^  {re.escape(job)}:\s*$", l):
+            start = i; break
+    if start is None:
+        sys.exit(f"ci-env: job '{job}' not found in {path}")
+    end = len(lines)
+    for j in range(start + 1, len(lines)):
+        if re.match(r"^  \S", lines[j]):
+            end = j; break
+    entries = {}  # name -> ("secret", source) | ("var", source) | ("literal", value)
+    in_env, env_indent = False, 0
+    for l in lines[start:end]:
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)